Behavioral.Automation.Template/Behavioral.Automation.Template.Bindings/AutomationConfig.json

```diff
@@ -0,0 +1,14 @@
+﻿{
+  "BASE_URL": "http://localhost:4200/",
+  "TEST_EMAIL": "",
+  "TEST_PASSWORD": "",
+  "BASE_AUTH_URL": "",
+  "BROWSER_PARAMS": "--window-size=1920,1080",
+  "ACCESS_CLIPBOARD": false, 
+  "DOWNLOAD_PATH": "",
+  "SEARCH_ATTRIBUTE": "automation-id",
```

Comment:
not `data-automation-id`?

Reply:
I can change to attribute name which we use, but it's really up to user how to name it